Index: src/utils/Validator.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package utils;\r\n\r\npublic class Validator {\r\n\r\n    public boolean ValidateNumberInput(String textNumber) {\r\n        try {\r\n            int number = Integer.parseInt(textNumber);\r\n        } catch (Exception e) {\r\n            return false;\r\n        }\r\n        return true;\r\n    }\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/utils/Validator.java b/src/utils/Validator.java
--- a/src/utils/Validator.java	(revision 6b9688fe66eb617607b6bf22eeadb83c3674d7a1)
+++ b/src/utils/Validator.java	(date 1627680151815)
@@ -4,7 +4,7 @@
 
     public boolean ValidateNumberInput(String textNumber) {
         try {
-            int number = Integer.parseInt(textNumber);
+            Integer.parseInt(textNumber);
         } catch (Exception e) {
             return false;
         }
